### DISCUSSION TITLE
Database

### DISCUSSION THREAD

**Discussion by Yoououuou** [iptv-org/community#329] [Q&A] [answered]
Why does the database show channels like wbz cbs Boston and wbts nbc Boston but when I download the us m3u, Massachusetts m3u, or make an m3u in the database, the onlyone that works is wcvb, abc Boston 5.

**Comment by BellezaEmporium** [comment, marked as answer]
*sigh*

### I've seen this channel on the website/the EPG, though it is not seen in the playlists, why ?
The website is a database of channels that are available worldwide, and the EPG repo scrapes the guides for various channels available on different websites.

Both these different scopes do **not** represent a list of streams that are available in the playlists, as they're entirely different repositories and serve different purposes.
